packages/font-color-ui/package.json

```diff
@@ -30,19 +30,17 @@
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "slate": "^0.60.0",
-    "slate-react": "^0.60.0"
+    "slate-react": "^0.60.0",
+    "slate-history": "^0.62.0"
   },
   "dependencies": {
     "@babel/polyfill": "^7.12.1",
     "@dreifuss-wysiwyg-editor/font-color": "0.0.0",
-    "@udecode/slate-plugins-core": "^1.0.0-next.36",
-    "@udecode/slate-plugins-toolbar": "^1.0.0-next.36",
-    "@dreifuss-wysiwyg-editor/common": "^1.0.2-alpha.1",
-    "@udecode/slate-plugins-ui-fluent": "^1.0.0-next.36"
+    "@udecode/plate-core": "^1.0.0",
+    "@dreifuss-wysiwyg-editor/common": "^1.0.2-alpha.1"
   },
   "devDependencies": {
     "@types/react": "^17.0.14",
-    "@types/react-dom": "^17.0.9",
```

Comment:
if it's in the  `peerDependencies` then the `@types` are needed here.

Reply:
fixed